docs/examples/sock-shop.md

```diff
@@ -167,31 +200,22 @@ Examining this command, we see two new arguments:
 gitops add app \
 --url ssh://git@github.com/example/microservices-demo.git \
 --path ./deploy/kubernetes/helm-chart \
---app-config-url NONE \
+--app-config-url ssh://git@github.com/example/external.git
 --deployment-type helm \                   # (1)
 --helm-release-target-namespace sock-shop  # (2)
+--auto-merge
 ```
 
 1. Since we're pulling the chart from a git repository, we need to explicitly state that we're using a helm chart. If we were using a helm repository, we would use `--chart <chart name>` instead of `--path <path to application>` and the deployment type would be unambiguous
 2. The application will be deployed in the namespace specified by `--helm-release-target-namespace`
 
```

Comment:
I feel like we need to close this off... maybe showing the status of the helm release?

Reply:
Interestingly, despite being able to access the application, querying helm says that the status is `failed` due to max install retries

Reply:
I see that too. Will check it out

Reply:
Closed off the section with "You can check the status of the application by running the `gitops get app microservices-demo` command."